app/src/main/res/layout/viewstub_visible_message_status_container.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this file used anywhere?

Reply:
Good spotting. I abandoned this file mid change.